src/commands/timeout.ts

```diff
@@ -0,0 +1,162 @@
+//Template: https://github.com/Costpap/CostBot/blob/main/src/commands/kick.ts
+
+//imports
```

Comment:
I don't see a need for such comments, please remove them in order to be consistent with the rest of the codebase.

```suggestion
```